addons/mail/controllers/discuss/rtc.py

```diff
@@ -88,6 +88,13 @@ def channel_call_leave(self, channel_id, session_id=None):
         # sudo: discuss.channel.rtc.session - member of current user can leave call
         member.sudo()._rtc_leave_call(session_id)
 
+    @http.route("/mail/rtc/channel/upgrade", methods=["POST"], type="jsonrpc", auth="user")
+    def channel_upgrade(self, channel_id):
+        member = request.env["discuss.channel.member"].search([("channel_id", "=", channel_id), ("is_self", "=", True)])
+        if not member:
+            raise NotFound()
+        member.sudo()._join_sfu(force=True)
+
```

Comment:
`upgrade` is too obscure, I would use a more specific name so it's more clear what's the purpose of this route, e.g. `upgrade_connection` or `upgrade_to_sfu`.